app-devel/judy/autobuild/prepare

```diff
@@ -0,0 +1,3 @@
+abinfo "Running configure ..."
+"${SRCDIR}"/bootstrap
+"${SRCDIR}"/configure --prefix="${PKGDIR}"/usr --enable-64-bit
```

Comment:
What? So is this an autotools source?

Reply:
`autotools` build type is used instead of `self` now.